### ISSUE TITLE
Missing settingstemplate.json file.

### ISSUE THREAD

**Issue by tmulz** [tjlundstrom/discord-transcription-bot#1]
```
{
"bot_controller" : "123456789",
"discord_token" : "abcdefghijklmnopqrstuvwxyz",
"content_type" : "audio/?????",
"watson_username" : "abcdefg",
"watson_password" : "hijklmnop",
"member_id_aa" : "10987654321",
"member_id_ab" : "10987654321",
"member_id_ac" : "10987654321",
"member_id_ad" : "10987654321",
"prefix" : "!"
}
```

**Comment by t3mulligan** [comment]
```
{
"bot_controller" : "123456789",
"discord_token" : "abcdefghijklmnopqrstuvwxyz",
"content_type" : "audio/?????",
"watson_username" : "abcdefg",
"watson_password" : "hijklmnop",
"member_id_aa" : "10987654321",
"member_id_ab" : "10987654321",
"member_id_ac" : "10987654321",
"member_id_ad" : "10987654321",
"prefix" : "!"
}
```

**Comment by millenniumdrummer** [comment]
An updated version of settingstemplate.json has been pushed.  I am guessing you were mostly interested in the "content_type".  As a friendly heads up, the "rate", i.e. sampling rate/frequency, is a required parameter when using audio/l16.  It seemed that the higher the bit rate from the original 44100 Hz that was used seemed to get Watson to have better accuracy, and lower it would cause Watson to have lower accuracy.  Of course, once at 88200 Hz, any higher seemed to drop off Watson's accuracy as the sampling rate went up.  I cannot say that my method is science and that I am close to even understanding the sampling rate system, but this is what I observed.  I did try up to 192k and down to 16k, but the results were not favorable.  Any rates between those and the 44.1k to 114k rates I tested are not verified.  I hope this helps.
